.circleci/config.yml

```diff
@@ -10,6 +10,10 @@ jobs:
     executor:
       name: hmpps/java
       tag: "21.0"
+    environment:
```

Comment:
Duplicated the memory settings we previously set on "from-nomis-migration" as build continually failed